Sources/LinuxPlatform/Linux.swift

```diff
@@ -141,5 +141,76 @@ public struct Linux: Platform {
         FileManager.default.temporaryDirectory.appendingPathComponent("swiftly-\(UUID())")
     }
 
+    public func verifySignature(httpClient: SwiftlyHTTPClient, archiveDownloadURL: URL, archive: URL) async throws {
+        guard (try? self.runProgram("gpg", "--version", quiet: true)) != nil else {
+            SwiftlyCore.print("gpg not installed, skipping signature verification.")
+            return
+        }
+
+        let foundKeys = (try? self.runProgram(
+            "gpg",
+            "--list-keys",
+            "swift-infrastructure@forums.swift.org",
+            "swift-infrastructure@swift.org",
+            quiet: true
+        )) != nil
+        guard foundKeys else {
+            SwiftlyCore.print("Swift PGP keys not imported, skipping signature verification.")
+            SwiftlyCore.print("To enable verification, import the keys from https://swift.org/keys/all-keys.asc")
```

Comment:
Can we provide help on how to do that. eg Provide `wget -q -O - https://swift.org/keys/all-keys.asc |
  gpg --import -`

Reply:
Good idea, done.